.circleci/config.yml

```diff
@@ -58,4 +58,5 @@ jobs:
       - checkout
       - run:
           name: Cargo audit
-          command: cargo audit
+          # RUSTSEC-2021-0078/79 unlikely to affect clients
+          command: cargo audit --ignore RUSTSEC-2021-0078 --ignore RUSTSEC-2021-0079
```

Comment:
How about definining this in `.cargo/audit.toml`? https://github.com/mozilla-services/merino/blob/main/.cargo/audit.toml